frontend/pages/blog/introducing-journaly.tsx

```diff
@@ -0,0 +1,295 @@
+import LandingPageLayout from '../../components/Layouts/LandingPageLayout'
+import BlogPageLayout from '../../components/Layouts/BlogPageLayout'
+
+const IntroducingJournalyBlogPost = () => (
+  <LandingPageLayout>
+    <BlogPageLayout
+      title="Introducing Journaly"
+      date="2019-09-25"
+      image={'../images/blog/post01-image.jpg'}
+    >
+      <p>
+        Hi there! I'm Robin. Long-time language enthusiast,{' '}
+        <a href="https://youtube.com/user/TheLifeOfRob">polyglot YouTuber</a>,
+        language teacher, and author of the book{' '}
+        <em>
+          <a href="https://www.kumabrand.com/shop">How To Maintain Languages</a>
+        </em>
```

Comment:
HOPEFULLY when this launches, we'll be able to change this to `robinmacpherson.co/shop` 🤩 hehe

Reply:
wooo!!!!